cmd/snap-confine/ns-support.c

```diff
@@ -73,6 +73,11 @@ static const char *sc_ns_dir = SC_NS_DIR;
  **/
 #define SC_NS_MNT_FILE ".mnt"
 
+enum {
+	HELPER_CMD_EXIT,
```

Comment:
```
HELPER_CMD_EXIT = 0,
```

Reply:
I would have done something like that if there was a public protocol but since this is all inside one binary we can reorder or shuffle those around without consequences.